parser/include/parser/parsercontext.h

```diff
@@ -14,24 +25,29 @@ namespace parser
 
 class SourceManager;
 
+ /**
+ * Defines file status categories for incremental parsing.
+ */
+enum class IncrementalStatus
+{
+  MODIFIED,
```

Comment:
What do these values mean?

Reply:
I think what's missing here is that this thing is "state in database VERSUS state on disk at parse time". The individual enumconstants are clean enough.

Reply:
Added in 7100014093aed0a1ae46a02ef6c1bff82a641139.